src/librustc_session/session.rs

```diff
@@ -593,6 +593,14 @@ impl Session {
         }
     }
 
+    pub fn must_emit_unwind_tables(&self) -> bool {
+        if let Some(x) = self.opts.cg.force_unwind_tables {
```

Comment:
As implemented, this allows users to turn off unwinding tables even if the target strictly requires them. That seems like a footgun.

Reply:
should we just error in case of such a conflict?

Reply:
Emitting an error has prior art in rustc so, yes.

Reply:
Ok, Will do when I get back to my work computer.

Reply:
I believe that I've done this in the right place: `validate_commandline_args_with_session_available`. This check only takes into account the target definition, not the `no_landing_pads` that is checked at attribute insertion time.